keys.go

```diff
@@ -9,6 +9,7 @@ type Key int
 const (
 	IPROTO_REQUEST_TYPE Key = 0x00
 	IPROTO_SYNC         Key = 0x01
+
```

Comment:
You could delete the empty lines.

Reply:
Those empty lines are present in the original file.
https://github.com/tarantool/tarantool/blob/05751e6c6869579446b34ae67e0605a68fc56b89/src/box/iproto_constants.h#L80-L112

Reply:
Updated.